igv_remote/_core.py

```diff
@@ -136,13 +130,16 @@ def _goto_multiple(self, expand=20, **kwargs):
         self._send("goto {}".format(" ".join(positions)))
 
     def _snapshot(self):
+        assert self._image_fulldir is not None, "Please set view optins with ir.set_saveopts() first"
```

Comment:
This should be an exception. Assertions are for developers to catch programming errors; if the code is written properly, an end user should never be able to fail an assertion, no matter what they input. On the other hand, exceptions are for communicating input errors to the end user.

Since it's easy for an end user to have `self._image_fulldir` equal to none, this should be an exception.